roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/Car.java

```diff
@@ -2,5 +2,63 @@
 
 import at.fhv.itm3.s2.roundabout.api.entity.ICar;
 
+import java.time.LocalTime;
+import java.util.List;
+
 public class Car implements ICar {
+
+    private LocalTime lastUpdateTime;
```

Comment:
why LocalTime? i know its from the entity diagram, but isn't it the simulation time (integer or long)?

Reply:
jep, that was the reason, i think it was double